worlds/witness/options.py

```diff
@@ -8,19 +8,23 @@
 
 
 class DisableNonRandomizedPuzzles(Toggle):
-    """Disables puzzles that cannot be randomized.
+    """
+    Disables puzzles that cannot be randomized.
     This includes many puzzles that heavily involve the environment, such as Shadows, Monastery or Orchard.
+
     The lasers for those areas will activate as you solve optional puzzles, such as Discarded Panels.
-    Additionally, the panels activating Monastery Laser and Jungle Popup Wall will be on from the start."""
+    Additionally, the panel activating the Jungle Popup Wall will be on from the start.
+    """
     display_name = "Disable non randomized puzzles"
 
 
 class EarlyCaves(Choice):
-    """Adds an item that opens the Caves Shortcuts to Swamp and Mountain,
-    allowing early access to the Caves even if you are not playing a remote Door Shuffle mode.
-    You can either add this item to the pool to be found on one of your randomized checks,
-    or you can outright start with it and have immediate access to the Caves.
-    If you choose "add_to_pool" and you are already playing a remote Door Shuffle mode, this setting will do nothing."""
+    """
+    Adds an item that opens the Caves Shortcuts to Swamp and Mountain, allowing early access to the Caves even if you are not playing a remote Door Shuffle mode.
+    You can either add this item to the pool to be found on one of your randomized checks, or you can outright start with it and have immediate access to the Caves.

```

Comment:
This implied that it was forced local, this might be a better wording:
```suggestion
    You can either add this item to the pool to be found in the multiworld, or you can outright start with it and have immediate access to the Caves.
```